moped-database/migrations/1734717637542_update_workgroups/down.sql

```diff
@@ -0,0 +1,3 @@
+-- Updating moped_workgroup table will be up only. If we need to revert, we will need do it manually or
+-- update with a future migration.
+SELECT 0;
```

Comment:
I like [what you've got](https://github.com/cityofaustin/atd-moped/blob/ea5ed9e8683f3f62c437fadab4476072d50fb2d6/moped-database/migrations/1735340549358_add-funding-status-description/down.sql), Mateo. It reverts back to the previous DB state cleanly ✅ 🚢

Reply:
+1 - i like the down migration for the funding descriptions too. i've been omitting down migrations for these reorg changes since they update records in other tables and would take some heavier lifting to go back.